book/_templates/layout.html

```diff
@@ -0,0 +1,13 @@
+{% extends "sphinx_book_theme/layout.html" %}
```

Comment:
can remove the entire `_templates` folder, this was a temporary fix for enabling google analytics